plugins/amazonq/mynah-ui/src/mynah-ui/connectorAdapter.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {ChatPrompt, MynahUI, QuickActionCommand, QuickActionCommandGroup} from '@aws/mynah-ui-chat'
+import { isTabType } from './ui/storages/tabsStorage'
+import { WebviewUIHandler } from './ui/main'
+import { TabDataGenerator } from './ui/tabs/generator'
+import { ChatClientAdapter, ChatEventHandler } from '@aws/chat-client'
+import { FqnExtractor } from "./fqn/extractor";
+
+export * from "./ui/main";
+
+declare global {
+    interface Window { fqnExtractor: FqnExtractor; }
+}
+
+window.fqnExtractor = new FqnExtractor();
+
+export const initiateAdapter = (showWelcomePage: boolean,
```

Comment:
can you provide reference to VSC?

Reply:
This doesn't have a reference in VSC. The webpack bundles this file and to be able to access its contents need an instance of the class